WordPress/Jetpack/Resources/AppStoreStrings.po

```diff
@@ -40,8 +40,6 @@ msgstr ""
 
 msgctxt "v20.2-whats-new"
 msgid ""
-"We updated the Insights view in your site’s Stats. See how your site is performing with a quick glance—and comparing performance to previous time periods is just a finger-tap away.\n"
-"\n"
```

Comment:
Hi @mokagio, I updated the What's New string here to accompany the removal of Stats Revamp from Jetpack's release notes. 
I'm not familiar with this part of the process, so please let me know if this is all that's needed.